apps/accounts/templates/provider_portal/provider_domain_new/step_1.html

```diff
@@ -7,49 +7,62 @@ <h3 class="border-none">Domain: </h3>
 
 
 {%  block current_step_pre %}
-<h3>Step 2: Create carbon.txt</h3>
+<h3>Step 2: Link Domain to a Carbon.txt file</h3>
+{% if provider.primary_linked_domain %}
 <p>
-The carbon.txt standard allows publishing sustainability data on the web in a way that is open, discoverable, comparable	and traceable. We use carbon.txt to etc etc etc.
-</p><p>
-You can either create a new carbon.txt file or point to the carbon.txt at an existing domain, using HTTP or DNS. More copy on the circumstances in which you might use each one.
+You can either create a new Carbon.txt file for {{preview_data.0.domain}} or link it to the Carbon.txt file hosted on your primary domain ({{provider.primary_linked_domain.domain}}) using DNS or HTTP Headers.
 </p>
-{% if provider.primary_linked_domain %}
+
+<p>Which method should you use?</p>
+
+<ul>
+    <li><b>Use a new Carbon.txt file if</b> you are able to upload content onto the server for {{preview_data.0.domain}}, <b>and if</b> the sustainability disclosures for {{preview_data.0.domain}} differ from those of your primary domain ({{provider.primary_linked_domain.domain}}).</li>
+    <li><b>Use DNS to link to an existing Carbon.txt file if</b> your are able to edit the DNS records for {{preview_data.0.domain}}, and you want this domain to share the same sustainability disclosures as your primary domain.</li>
+    <li><b>Use HTTP Headers to link to an existing Carbon.txt file if</b> your are able to edit the server configuration for {{preview_data.0.domain}} to return additional response headers, and you want this domain to share the same sustainability disclosures as your primary domain.</li>
+</ul>
+
+
 <div class="tabbedview my-5 py-5">
   <ul role="tablist" class="tablist flex flex-wrap text-sm font-medium text-center list-none w-full justify-items-stretch pl-0 m-0">
       <li class="pl-0 m-0 grow">
-          <a href="#" role="tab" aria-selected="true" aria-controls="carbontxt-file-panel" id="carbontxt-file-tab" class="tab inline-block p-4 border rounded-tl-lg w-full active m-0 no-underline hover:underline">Create a new carbon.txt</a>
+          <a href="#" role="tab" aria-selected="true" aria-controls="carbontxt-file-panel" id="carbontxt-file-tab" class="tab inline-block p-4 border rounded-tl-lg w-full active m-0 no-underline hover:underline">Create a new Carbon.txt file</a>
       </li>
       <li class="pl-0 m-0 grow">
-          <a href="#" role="tab" aria-selected="false" aria-controls="carbontxt-dns-panel" id="carbontxt-dns-tab" class="tab inline-block p-4 border-y bg-neutral-200 hover:bg-neutral-100 w-full m-0 no-underline hover:underline">Use existing with DNS</a>
+          <a href="#" role="tab" aria-selected="false" aria-controls="carbontxt-dns-panel" id="carbontxt-dns-tab" class="tab inline-block p-4 border-y bg-neutral-200 hover:bg-neutral-100 w-full m-0 no-underline hover:underline">Use existing Carbon.txt with DNS</a>
```

Comment:
Hmm, the extra width introduced here has slightly broken the layout of that tabbed box for me: 
![image](https://github.com/user-attachments/assets/4e9991b3-8cba-4141-bae1-7ae6085d0808)
(Firefox, 1920x1080) - I can take a look at this if needed.